openfire/pom.xml

```diff
@@ -12,10 +12,25 @@
     <artifactId>jitsi-videobridge-openfire</artifactId>
     <version>2.0-SNAPSHOT</version>
     <name>Jitsi Videobridge Openfire Plugin</name>
+    <description>Integrates Jitsi Video Bridge into Openfire.</description>
+
+    <organization>
+        <name>Jitsi</name>
+        <url>https://jitsi.org</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
 
     <properties>
         <!-- Override the default plugin name, as the default include illegal characters. -->
         <plugin.name>jitsivideobridge</plugin.name>
+        <videobridge.version>1.0-20170523.204532-356</videobridge.version>
```

Comment:
The only alternative that I can think if is to depend on -SNAPSHOT, which on one hand would automatically allow us to follow the changes in videobridge, but on the other hand... well, make use automatically follow the changes in videobridge - even when they introduce breaking changes. :) By pinning the dependency to a specific version, we prevent that.